Dockerfile

```diff
@@ -300,7 +287,7 @@ COPY --from=swagger       /build/ /usr/local/bin/
 COPY --from=tomlv         /build/ /usr/local/bin/
 COPY --from=tini          /build/ /usr/local/bin/
 COPY --from=registry      /build/ /usr/local/bin/
-COPY --from=criu          /build/ /usr/local/
+COPY --from=criu          /usr/bin/criu /usr/local/bin
```

Comment:
CI probably fails because this needs to be /usr/sbin/criu

Reply:
I tested this patch (with `make BIND_DIR=. shell`) and it works with the following change:
```diff
-COPY --from=criu          /usr/bin/criu /usr/local/bin
+COPY --from=criu          /usr/sbin/criu /usr/local/bin
```

Reply:
Ah, so stupid of me; fixed now.